.nvmrc

```diff
@@ -0,0 +1 @@
+v16.19.0
```

Comment:
Node.js 18 is the active line at the moment. See https://nodejs.dev/en/about/releases/.

Reply:
I went with 16 because that's what's listed in `package.json`:

https://github.com/WordPress/wordpress-playground/blob/9c7d797f7eacc56ead54e80d0f6a1e5282d425d9/package.json#L61-L64

Reply:
Right, but it doesn't prevent using v18. It only sets the minimum version.

Reply:
Ok. I don't have a preference on which version is used.

Reply:
It's up to @adamziel then, it can be Node 16, too 😄

Reply:
I'd go with Node 18 just because it's an LTS release

Reply:
@adamziel Sounds good, updated.